### DISCUSSION TITLE
DOFs for the IEA 15MW floater model

### DISCUSSION THREAD

**Discussion by QusayHawari** [NatLabRockies/ROSCO#137] [Q&A]
Hi,

I am working on the 15MW semi-sub floater model in closed loop operation using a turbulent speed with a mean of 20m/s. I am also using a 5m wave elevation with a period of 10s in the Hyrdrodyn file. 

The DOFs used are shown below:

![DOFs_working](https://user-images.githubusercontent.com/91909348/169866509-d44c6522-e2c7-4f0f-a198-be4834ca7bf3.PNG)

I get the following closed loop response for the generator speed and actuation commands:

![CL_GS_BLD1PIT_Works](https://user-images.githubusercontent.com/91909348/169866596-c75bd6a4-55d4-4724-93e6-6922d3b95953.jpg)

The simulation seems to work fine (please correct me if you see the simulation does not look right :) ).

I am facing a problem where any extra DOF I turn on (except for yaw and teeter) the simulation does not run (fatal) with the following message:

![Fatat_Error](https://user-images.githubusercontent.com/91909348/169866932-0b0aad29-d4e4-4dbd-9844-f843a593c2c5.PNG)

Any reason why this is happening?

Please let me know if you need any further details about the simulation.

Thanks in advance,
Qusay.

**Comment by dzalkind** [comment]
This looks like some issues we were having with the 2nd tower modes in previous versions of the IEA-15MW model.  Can you confirm you are using the lastest version of that model?  https://github.com/IEAWindTask37/IEA-15-240-RWT

**Reply by QusayHawari** [reply]
Hi,

Yes, I was using the latest versions of the turbine model, OpenFAST, and ROSCO. 

I also double checked and downloaded the model in the link you sent, and the error still exists. 

This is the folder I am using. 

[IEA_15MW_Latest.zip](https://github.com/NREL/ROSCO/files/8756929/IEA_15MW_Latest.zip)

Thanks in advance,
Qusay.
